### PR TITLE
Fix: Discord.Discord version 1.0.9144

### PR DESCRIPTION
**Pull request by seangibbz**
Checklist for Pull Requests
- [Y] Have you signed the [Contributor License Agreement](https://cla.opensource.microsoft.com/microsoft/winget-pkgs)?
- [N] Is there a linked Issue?
	- https://github.com/microsoft/winget-pkgs/issues/137934
	- https://github.com/microsoft/winget-pkgs/issues/137157
	- https://github.com/microsoft/winget-pkgs/issues/136960

Manifests
- [Y] Have you checked that there aren't other open [pull requests](https://github.com/microsoft/winget-pkgs/pulls) for the same manifest update/change?
- [Y] This PR only modifies one (1) manifest
- [Y] Have you [validated](https://github.com/microsoft/winget-pkgs/blob/master/doc/Authoring.md#validation) your manifest locally with `winget validate --manifest <path>`?
- [Y] Have you tested your manifest locally with `winget install --manifest <path>`?
- [Y] Does your manifest conform to the [1.6 schema](https://github.com/microsoft/winget-pkgs/tree/master/doc/manifest/schema/1.6.0)?

Note: `<path>` is the name of the directory containing the manifest you're submitting.

---

 ###### Microsoft Reviewers: [Open in CodeFlow](https://microsoft.github.io/open-pr/?codeflow=https://github.com/microsoft/winget-pkgs/pull/157519)

### CODE REVIEW

File: manifests/d/Discord/Discord/1.0.9144/Discord.Discord.locale.en-US.yaml [hunk 1]
```diff
@@ -1,23 +1,22 @@
...
-PrivacyUrl: https://discord.com/privacy/
+PrivacyUrl: https://discord.com/privacy
 Author: Discord Inc.
 PackageName: Discord
 PackageUrl: https://discord.com/download
 License: Proprietary
-LicenseUrl: https://discord.com/terms/
+LicenseUrl: https://discord.com/terms
 Copyright: Copyright (c) 2024 Discord Inc. All rights reserved.
+CopyrightUrl: https://support.discord.com/hc/articles/4410331.0.9341.0.9655
```
Comment (stephengillie) [inline manifests/d/Discord/Discord/1.0.9144/Discord.Discord.locale.en-US.yaml:14]:
```suggestion
# CopyrightUrl: 
```

File: manifests/d/Discord/Discord/1.0.9144/Discord.Discord.locale.en-US.yaml [hunk 2]
```diff
@@ -29,5 +28,8 @@ Tags:
 - voice-chat
 ReleaseNotesUrl: https://discord.com/tags/changelog
 PurchaseUrl: https://discord.com/nitro
+Documentations:
+- DocumentLabel: Support
+  DocumentUrl: https://support.discord.com/hc/en-us
```
Comment (stephengillie) [inline manifests/d/Discord/Discord/1.0.9144/Discord.Discord.locale.en-US.yaml:33]:
```suggestion
  # DocumentUrl: 
```

**Review by stephengillie — CHANGES_REQUESTED** [review]
The `/hc/en-us` usually indicates that the developer is using the ZenDesk Help Center, and unfortunately they've blocked our automated scans.

File: manifests/d/Discord/Discord/1.0.9144/Discord.Discord.locale.pt-BR.yaml [hunk 3]
```diff
@@ -26,5 +25,8 @@ Tags:
 - voz
 ReleaseNotesUrl: https://discord.com/tags/changelog
 PurchaseUrl: https://discord.com/nitro
+Documentations:
+- DocumentLabel: Support
+  DocumentUrl: https://support.discord.com/hc/en-us
```
Comment (stephengillie) [inline manifests/d/Discord/Discord/1.0.9144/Discord.Discord.locale.pt-BR.yaml:30]:
```suggestion
 # DocumentUrl: 
```

File: manifests/d/Discord/Discord/1.0.9144/Discord.Discord.locale.zh-CN.yaml [hunk 4]
```diff
@@ -1,21 +1,22 @@
...
-PrivacyUrl: https://discord.com/privacy/
+PrivacyUrl: https://discord.com/privacy
 Author: Discord Inc.
 PackageName: Discord
 PackageUrl: https://discord.com/download
 License: 专有软件
-LicenseUrl: https://discord.com/terms/
+LicenseUrl: https://discord.com/terms
 Copyright: Copyright (c) 2024 Discord Inc. All rights reserved.
+CopyrightUrl: https://support.discord.com/hc/articles/4410331.0.9341.0.9655
```
Comment (stephengillie) [inline manifests/d/Discord/Discord/1.0.9144/Discord.Discord.locale.zh-CN.yaml:14]:
```suggestion
# CopyrightUrl:
```

**Review by stephengillie — CHANGES_REQUESTED** [review]
https://github.com/microsoft/winget-pkgs/pull/157519#pullrequestreview-2113319944

**Comment by stephengillie** [comment]
> [#157519 (review)](https://github.com/microsoft/winget-pkgs/pull/157519#pullrequestreview-2113319944)

**Review by seangibbz — COMMENTED** [review]
hc urls removed

**Review by stephengillie — APPROVED** [review]

**Pull request merged**